src/client/src/orangehrmAdminPlugin/pages/localization/LocalizationConfiguration.vue

```diff
@@ -46,10 +46,11 @@
           <oxd-grid :cols="2" class="orangehrm-full-width-grid">
             <oxd-grid-item>
               <oxd-input-field
-                v-model="configuration.dateFormat"
```

Comment:
without removing this can't we select `Y-m-d` format. may be hard coding format under `created()`

Reply:
Oh ok will do that

Reply:
Fixed